NEWS

```diff
@@ -56,6 +56,9 @@ Unreleased (2.0.0)
   * `mongoc_index_opt_t`, `mongoc_index_opt_geo_t` `mongoc_index_opt_wt_t` are removed. Pass options using `bson_t` to `mongoc_collection_create_indexes_with_opts` instead.
   * `mongoc_collection_find_indexes` is removed. Use `mongoc_collection_find_indexes_with_opts` instead.
   * See [MongoDB documentation](https://www.mongodb.com/docs/languages/c/c-driver/current/indexes/) for working with indexes.
+* Deprecated API for `mongoc_database_t` has been removed:
+  * `mongoc_database_find_collections` is removed. Use `mongoc_database_find_collections_with_opts` instead.
+  * `mongoc_database_get_collection_names` is removed. Use `mongoc_database_get_collection_names` instead.
```

Comment:
"X is removed. Use X instead" Typo?

Reply:
Good spot. Fixed. The expected alternative is `mongoc_database_get_collection_names_with_opts`.